host/ColorBuffer.cpp

```diff
@@ -487,9 +511,11 @@ bool ColorBuffer::Impl::flushFromVkBytes(const void* bytes, size_t bytesSize) {
 }
 
 bool ColorBuffer::Impl::invalidateForGl() {
```

Comment:
flushFrom* and invalidateFor* functions should probably be inside GFXSTREAM_ENABLE_HOST_GLES, I'll have a look at this.